docs_user/modules/proc_pulling-configuration-from-a-tripleo-deployment.adoc

```diff
@@ -20,16 +26,16 @@ connection=ssh
 remote_config_path=/tmp/tripleo
 ----
 +
-Make sure the ssh command you provide in `ssh_cmd` parameter is correct and with key authentication.
+Ensure that the ssh command you provide in `ssh_cmd` parameter is correct and includes key authentication.
 
-. Enable or disable the services that you want in the `/etc/os-diff/config.yaml` file. Make sure that you have the correct rights to edit the file, for example:
+. Enable the services that you want to include in the `/etc/os-diff/config.yaml` file, and disable the services that you want to exclude from the file. Ensure that you have the correct permissions to edit the file, for example:
 +
 ----
-chown ospng:ospng /etc/os-diff/config.yaml
+$ chown ospng:ospng /etc/os-diff/config.yaml
```

Comment:
Can we remove "for example:" and just leave this at "Ensure that you have the correct permissions to edit the file:" ?

Reply:
Yes